.gitignore

```diff
@@ -32,6 +32,7 @@ dockstore-client/target/
 swagger-java-client/target/
 swagger-java-quay-client/target/
 swagger-java-sam-client/target/
+swagger-java-bitbucket-client/target/
```

Comment:
That's odd, is this changing in this PR or just forgotten previously?

Reply:
It is previously forgotten.